app/react/Favorites/components/specs/FavoriteBanner.spec.tsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import React from 'react';
+import { shallow, ShallowWrapper } from 'enzyme';
+
+import { FavoriteBanner, FavoriteBannerProps } from '../FavoriteBanner';
+
+const localStorageMock = (() => {
+  let store: any = {};
+  return {
+    getItem: (key: string) => store[key],
+
+    setItem: (key: string, value: string) => {
+      store[key] = value.toString();
+    },
+
+    clear: () => {
+      store = {};
+    },
+
+    removeItem: (key: string) => {
+      delete store[key];
+    },
+  };
+})();
+
+Object.defineProperty(window, 'localStorage', { value: localStorageMock });
+
+describe('Favorite Banner', () => {
+  let component: ShallowWrapper;
+  let mockEvent: any;
+
+  const render = (props: FavoriteBannerProps = { sharedId: 'sharedId2' }) => {
+    component = shallow(<FavoriteBanner {...props} />);
+  };
+
+  beforeEach(() => {
+    localStorageMock.setItem('uwaziFavorites', 'sharedId1,sharedId4');
+    mockEvent = jasmine.createSpyObj('mockEvent', ['stopPropagation', 'preventDefault']);
+  });
+
+  it('should present a clickable banner', () => {
+    render();
+    expect(component).toMatchSnapshot();
+  });
```

Comment:
@RafaPolit we should avoid full component snapshots or snapshots at all, i do not think this has any value, do not reflect any intent, in some cases are too big to debug and usually test implementation details like the label in this case that is not relevant to the functionality.

Reply:
Mostly agree.  Everything of this component was already tested afterwards, so there was absolutely no need for this test.  Removed it and extended the selectors that the 'required' classes are included.